volatility/framework/plugins/windows/dlllist.py

```diff
@@ -86,23 +89,34 @@ def _generator(self, procs):
                                                                 "pe",
                                                                 class_types = pe.class_types)
 
+        kuser = info.Info.get_kuser_structure(self.context, self.config['primary'], self.config['nt_symbols'])
+        nt_major_version = int(kuser.NtMajorVersion)
+        nt_minor_version = int(kuser.NtMinorVersion)
+        # this only applies to versions higher or equal to Window 7 (6.1 and higher)
+        dll_load_time_field = (nt_major_version > 6) or (nt_major_version == 6 and nt_minor_version >= 1)
         for proc in procs:
 
             proc_id = proc.UniqueProcessId
             proc_layer_name = proc.add_process_layer()
 
             for entry in proc.load_order_modules():
 
-                BaseDllName = FullDllName = renderers.UnreadableValue()
+                BaseDllName = FullDllName = DllLoadTime = renderers.UnreadableValue()
                 try:
                     BaseDllName = entry.BaseDllName.get_string()
                     # We assume that if the BaseDllName points to an invalid buffer, so will FullDllName
                     FullDllName = entry.FullDllName.get_string()
                 except exceptions.InvalidAddressException:
                     pass
 
+                if dll_load_time_field:
+                    try:
+                        DllLoadTime = conversion.wintime_to_datetime(entry.LoadTime.QuadPart)
+                    except:
```

Comment:
What kind of an exception do we expect to catch?  `Except pass:` is generally bad form.

Reply:
I'm changing this to `AttributeError`, because version prior to 6.1 won't have the `LoadTime` attribute.
Also it seems that `LoadTime` is a `LARGE_INTEGER`, and if run on 32bit host there will be not `QuadPart `(`QuadPart `is used several times in volatility, mainly in `get_create_time()` functions)
Maybe we should calculate the `QuadPart` ourselves using `LowPart `and `HighPart`:
[MSDN](https://docs.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-large_integer-r1)